contracts/anchor/src/state.rs

```diff
@@ -30,10 +30,9 @@ pub const NULLIFIERS: Map<Vec<u8>, bool> = Map::new("used_nullifers");
 //       `chain_id`(blockchain-unique ID) obtained inside the contract.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct Anchor {
-    pub chain_id: u64,           // ChainID of underlying blockchain(Temporary field)
-    pub handler: Addr,           // Address of "anchor-handler", which add/updte the `edge` info
-    pub proposal_nonce: u32,     // Proposal nonce
-    pub deposit_size: Uint128,   // Minimum `deposit` amount for tx
+    pub handler: Addr, // Address of "anchor-handler", which add/updte the `edge` info
```

Comment:
Can we put comments above the code? On the side is bad style.

Reply:
This should happen for all of these parameters and please use three slashes `///`.

Reply:
Done https://github.com/webb-tools/protocol-cosmwasm/pull/57/commits/ac2759ac676c3702d19f579ca740bb36e18481ef